src/getRepoInfo.py

```diff
@@ -0,0 +1,12 @@
+from github import Github
+import os
+
+#using a token for connection
+token = os.getenv('MY_TOKEN')
```

Comment:
What do you think about naming the env var as GITHUB_TOKEN?

Reply:
It's more suitable.